docs/reference/modules.rst

```diff
@@ -29,3 +29,4 @@ Utilities
 
    configuration
    logging
+   utils
```

Comment:
Careful here: We have always considering functions in `utils` implementation details that can change at any time. If we now make all the functionality there "public" (documented) we would lose that ability to do so.

Have you considered other places for the added functionality? Or defining manually which functions are documented (and thus supported) publicly?

Reply:
Should we rename `utils` to `_utils` to indicate that it is non public?

> Or defining manually which functions are documented (and thus supported) publicly?

I tried that because I didn't like that each util module is documented separately. But autosummary is pretty inflexible and doing this is kind of awkward.

> Have you considered other places for the added functionality?

We could add `bin_edges` at the top level. We were considering a bin-edge toolkit at some point anyway. Although, we put that on ice for now, right?

Reply:
I think `midpoints` is top-level?

If we follow my thoughts below (making this a method of the coords class) then the problem vanishes anyway?